efficientdet/keras/train_lib.py

```diff
@@ -239,7 +247,8 @@ def get_callbacks(params, profile=False):
       params['model_dir'], verbose=1, save_weights_only=True)
   early_stopping = tf.keras.callbacks.EarlyStopping(
       monitor='val_loss', min_delta=0, patience=10, verbose=1)
-  return [tb_callback, ckpt_callback, early_stopping]
+  display_callback = DisplayCallback('./testdata/img1.jpg', os.path.join(params['model_dir'], 'train'))
```

Comment:
If user doesn't have ./testdata/img1.jpg,  would the training fail?
Maybe we should check if this file doesn't exist, then give a warning and continue training without display callback?